tests/models/phi/test_modeling_phi.py

```diff
@@ -417,18 +417,18 @@ def test_model_phi_1_logits(self):
 
         self.assertTrue(torch.allclose(EXPECTED_OUTPUT, output[0, :2, :30], atol=1e-4, rtol=1e-4))
 
```

Comment:
I the following test no longer valid or should it be updated?

Reply:
Just re-enabled it, I commented out just to prevent any issue while I was updating our phi-1.5 repository with the internal code.